eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -102,7 +133,7 @@ jobs:
         condition: succeededOrFailed()
         displayName: "Publish Report Artifacts"
         inputs:
-          artifactName: reports
+          artifactName: lintReports
```

Comment:
@mikeharder Had to change this artifact name, because it complains if I publish to the same artifact name twice. It doesn't append all the files as we suspected.

Reply:
I'd actually suggest renaming your artifact from reports to "package-diffs" instead of renaming this one.

Reply:
done